zhaquirks/philips/__init__.py

```diff
@@ -218,3 +218,24 @@ def send_press_event(click_count):
         else:
             action = f"{button}_{press_type}"
             self.listener_event(ZHA_SEND_EVENT, action, event_args)
+
+
+class PhilipsEffectCluster(CustomCluster):
+    """Philips effect cluster."""
+
+    cluster_id = 0xFC03
+    ep_attribute = "philips_effect"
+
+    server_commands = {
+        0x00: foundation.ZCLCommandDef(
+            "set_effect",
+            schema={
+                "param1": t.uint8_t,
```

Comment:
do we know what these params represent?

Reply:
As far as I know they were just snooped from the Hue gateway. I've commented on the original issue where I found some additional behavior. The param1 seems to act like some kind of a command ID, and then the rest of params behave different based on that, but that's as much as I managed to observe.